### DISCUSSION TITLE
New Driver for FPrime

### DISCUSSION THREAD

**Discussion by Nainikas** [nasa/fprime#2863] [General]
Hello!
I am trying to do a project with an RC kit with RPI 4 integrated with FPrime. How do I proceed about making a new driver for FPrime to edit my I2C connections to a new board?
Thank you!

**Comment by LeStarch** [comment]
We have a linux I2C driver that will handle the bus protocol.  You would want to build the manager layer as documented here: https://nasa.github.io/fprime/UsersGuide/best/app-man-drv.html
